datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts

```diff
@@ -47,23 +52,28 @@ export function filterSchemaRows(
         entityRegistry,
         formattedFilterText,
     );
+
     const finalFieldPaths = new Set();
     const expandedRowsFromFilter = new Set();
 
     rows.forEach((row) => {
         if (
             matchesFieldName(row.fieldPath, formattedFilterText) ||
-            matchesEditableTagsOrTerms(row, filteredFieldPathsByEditableMetadata) ||
-            matchesTagsOrTerms(row, formattedFilterText, entityRegistry) // non-editable tags and terms
+            matchesEditableTagsOrTermsOrDescription(row, filteredFieldPathsByEditableMetadata) ||
+            matchesEditableTagsOrTermsOrDescription(row, filteredFieldPathsByEditableMetadata) ||
+            matchesTagsOrTermsOrDescription(row, formattedFilterText, entityRegistry) || // non-editable tags and terms
+            matchesDescription(row.description, formattedFilterText) // non-editable description
```

Comment:
so you don't need `matchesEditableTagsOrTermsOrDescription` twice, you can remove one of them.

Also you can remove `matchesDescription` since we're already checking for that in `matchesTagsOrTermsOrDescription`